addon/locales/en/translations.js

```diff
@@ -62,7 +62,7 @@ export default {
         },
         navbar: {
             add: 'Add',
-            addAPreprint: 'Add a {{preprintWords.Preprint}}',
+            addAPreprint: 'Add a {{documentType}}',
```

Comment:
Drive-by comment:

It might be good to leave the translations file in charge of whether it wants capital/plural, and pass the whole `documentType` into `t`:
```hbs
{{t 'navbar.addAPreprint' documentType=preprint.provider.documentType}}
```
```js
addAPreprint: 'Add a {{documentType.singularCapitalized}}',
```
That way if someone decides we want a lower-case `p`, the change only happens in one place.

Reply:
Excellent idea. Will do that.